api/tools/src/main/java/org/pmiops/workbench/tools/LoadInstitutions.java

```diff
@@ -0,0 +1,83 @@
+package org.pmiops.workbench.tools;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.util.logging.Logger;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.pmiops.workbench.institution.InstitutionMapperImpl;
+import org.pmiops.workbench.institution.InstitutionService;
+import org.pmiops.workbench.institution.InstitutionServiceImpl;
+import org.pmiops.workbench.institution.PublicInstitutionDetailsMapperImpl;
+import org.pmiops.workbench.model.Institution;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+@Configuration
+@Import({
+  InstitutionServiceImpl.class,
+  InstitutionMapperImpl.class,
+  PublicInstitutionDetailsMapperImpl.class
+})
+public class LoadInstitutions {
+
+  private static final Logger log = Logger.getLogger(LoadInstitutions.class.getName());
+
+  private static Option importFilename =
+      Option.builder()
+          .longOpt("import-filename")
+          .desc("File containing JSON for institutions to save")
+          .required()
+          .hasArg()
+          .build();
+  private static Option dryRunOpt =
```

Comment:
thinking out loud, but I wonder if it wouldn't be easier for the Ruby layer to build a JSON object of the options and just pass them to Java as one chunk. No need to change this now though.

Reply:
Hmm, that would make the gradle commands pretty unwieldy if they're used without project.rb.

Reply:
oh I didn't look closely. if you use those option vars elsewhere, nvm.